src/elm/Auth/Session.elm

```diff
@@ -46,16 +46,16 @@ type Session
 -- HELPERS
 
 
-{-| refreshAccessToken is a helper to schedule
-a job to try and refresh the access token
+{-| refreshAccessToken : is a helper to schedule
```

Comment:
nitpick but the grammar on some of these reads weird after we add the `:`

```suggestion
{-| refreshAccessToken : helper to schedule
```

Reply:
cleaned up these types of comments.